test/plots/piePlotTests.ts

```diff
@@ -2,92 +2,140 @@
-      piePlot.renderTo(svg);
-      let renderArea = (<any> piePlot)._renderArea;
-      let arcPaths = renderArea.selectAll(".arc");
-      let arcFillPaths = renderArea.selectAll(".arc.fill");
-      let arcOutlinePaths = renderArea.selectAll(".arc.outline");
-      assert.strictEqual(arcPaths.size(), 4, "2 paths per datum");
-      assert.strictEqual(arcFillPaths.size(), 2, "1 fill path per datum");
-      assert.strictEqual(arcOutlinePaths.size(), 2, "1 outline path per datum");
-      assert.strictEqual(arcFillPaths.style("stroke"), "none", "fill paths have no stroke");
-      assert.strictEqual(arcOutlinePaths.style("fill"), "none", "outline paths have no fill");
-      svg.remove();
-    });
+      beforeEach(() => {
+        svg = TestMethods.generateSVG(500, 500);
+        simpleData = [{value: 5}, {value: 15}];
+        simpleDataset = new Plottable.Dataset(simpleData);
+        piePlot = new Plottable.Plots.Pie();
+        piePlot.addDataset(simpleDataset);
+        piePlot.sectorValue((d) => d.value);
+        piePlot.renderTo(svg);
+      });
 
-    it("each entity selection consists of a fill path and a stroke path", () => {
-      let svg = TestMethods.generateSVG(500, 500);
-      let piePlot = new Plottable.Plots.Pie();
-      piePlot.sectorValue((d) => d.value);
-      let data = [
-        { value: 1 },
-        { value: 1 }
-      ];
-      let dataset = new Plottable.Dataset(data);
-      piePlot.addDataset(dataset);
-      piePlot.renderTo(svg);
-
-      let entities = piePlot.entities();
-      entities.forEach((entity) => {
-        assert.strictEqual(entity.selection.size(), 2, "each entity selection has 2 paths");
-        assert.strictEqual(entity.selection.filter(".fill").size(), 1, "each entity selection has 1 fill path");
-        assert.strictEqual(entity.selection.filter(".outline").size(), 1, "each entity selection has 1 stroke path");
+      it("draws a fill path and a stroke path for each slice", () => {
+        let arcPaths = piePlot.content().selectAll(".arc");
+        let arcFillPaths = piePlot.content().selectAll(".arc.fill");
+        let arcOutlinePaths = piePlot.content().selectAll(".arc.outline");
+        assert.strictEqual(arcPaths.size(), simpleData.length * 2, "2 paths per datum");
+        assert.strictEqual(arcFillPaths.size(), simpleData.length, "1 fill path per datum");
+        assert.strictEqual(arcOutlinePaths.size(), simpleData.length, "1 outline path per datum");
+        assert.strictEqual(arcFillPaths.style("stroke"), "none", "fill paths have no stroke");
+        assert.strictEqual(arcOutlinePaths.style("fill"), "none", "outline paths have no fill");
+        svg.remove();
       });
 
-      svg.remove();
-    });
+      it("draws slices proportional in angle to their value", () => {
+        let arcPaths = piePlot.content().selectAll(".arc.fill");
+        assert.lengthOf(arcPaths[0], 2, "only has two sectors");
+        let arcPath0 = d3.select(arcPaths[0][0]);
```

Comment:
I'm going to guess that the original author wanted to avoid confusion that might be caused by having the name-index and actual index be different.